### ISSUE TITLE
Check and fix content_reward and vesting_reward description in comments

### ISSUE THREAD

**Issue by timurkupaev** [GolosChain/golos#495]
https://github.com/GolosChain/golos/blob/master/libraries/chain/database.cpp#L2187
https://github.com/GolosChain/golos/blob/master/libraries/chain/database.cpp#L2190

**Comment by bitfag** [comment]
What about fixing the whole inflation comment description? It is outdated 100%

**Comment by afalaleev** [comment]
Yes, you're right, the description of inflation and constants should be fixed.

**Comment by afalaleev** [comment]
PR #548
